doc/2/guides/main-concepts/3-querying/index.md

```diff
@@ -8,3 +8,604 @@ order: 300
+
+Clauses can be used directly or composed with a [Boolean Query](/core/2/guides/main-concepts/3-querying#boolean-query).
+
+**Example:** _Simple top level clause: "city" field must be equal to "Antalya"_
+```js
+{
+  query: {
+    term: { city: "Antalya" }
+  }
+}
+```
+
+### Fake Data
+
+Throughout this guide we will use this set of documents to perform search queries:
+
+<details><summary>Create a collection and some documents</summary>
+
+```bash
+kourou collection:create ktm-open-data thamel-taxi '{
+  mappings: {
+    properties: {
+      city: { type: "keyword" },
+      name: { type: "keyword" },
+      age: { type: "integer" },
+      description: { type: "text" }
+    }
+  }
+}'
+
+kourou document:mCreate ktm-open-data thamel-taxi '{
+  documents: [
+    { 
+      _id: "aschen",
+      body: {
+        city: "Tirana",
+        name: "Aschen",
+        age: 27,
+        description: "Ruby is life"
+      }
+    },
+    { 
+      _id: "jenow",
+      body: {
+        city: "Tirana",
+        name: "Jenow",
+        age: 32,
+        description: "Java is my only love"
+      }
+    },
+    { 
+      _id: "liia",
+      body: {
+        city: "Kathmandu",
+        name: "Liaa",
+        age: 30,
+        description: "Little Princes is great"
+      }
+    },
+    { 
+      _id: "domisol",
+      body: {
+        city: "Siccieu",
+        name: "Dominique",
+        age: 61,
+        description: "I use to like PERL"
+      }
+    }
+  ]
+}'
+```
+
+</details>
+
+### `term` clause
+
+The [term](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-term-query.html) clause allows to returns documents that **contain an exact value** in a provided field.
+
+Term clause should be used on fields with the [keyword](/core/2/guides/main-concepts/2-data-storage#mappings-properties) type.
+
+::: info
+You can use the `term` clause to find documents based on a precise value such as a price, a product ID, or a username.
+:::
+
+**Example:** _Search for documents containing an exact field value_ 
+```bash
+kourou document:search ktm-open-data thamel-taxi '{
+  term: { name: "Jenow" }
+}'
+```
+
+::: info
+The search query content will be injected in the request body inside the `query` property:
+```js
+{
+  query: {
+    term: { name: "Jenow" }
+  }
+}
+```
+:::
+
+### `match` clause
+
+The [match](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-match-query.html) clause allows to returns documents **that match (could be approximately)** a provided `text`, `number`, `date` or `boolean` field.
+
+The match query is the standard query for **performing a full-text search**, including options for fuzzy matching.
+
+::: info
+The `match` clause to find documents with fields containing a value. The `match` clause as well as the content of a `text` fields are [analyzed](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/analysis-analyzers.html) by Elasticsearch before performing the query.
```

Comment:
I wanted to say that the Elasticsearch analyze the `match` clause value. For example it will separate the words, trim spaces, lowercase everything etc

That what they say in the documentation https://www.elastic.co/guide/en/elasticsearch/reference/7.4/query-dsl-match-query.html#query-dsl-match-query

Reply:
Well, then I think that the first sentence is confusing and that it doesn't bring value to the explanation. I think that this note is a lot better without it. wdyt?

```suggestion
The `match` clause as well as the content of a `text` fields are [analyzed](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/analysis-analyzers.html) by Elasticsearch before performing the query.
```

Reply:
Yes I agree too it's better. First I wanted to explain that it's not an exact match but I think the example did it better